cmd/wavefront-collector/main.go

```diff
@@ -260,20 +256,20 @@ func registerVersion() {
 	m.Update(f)
 }
 
-func createSourceManagerOrDie(src flags.Uris, scrapeTimeout time.Duration) metrics.MetricsSource {
-	sourceFactory := sources.NewSourceFactory()
-	sourceList := sourceFactory.BuildAll(src)
+// func createSourceManagerOrDie(src flags.Uris, scrapeTimeout time.Duration) metrics.MetricsSource {
```

Comment:
clean up the commented parts. Retain this method and log fatal if `NewSourceManager` returns an error.